controllers/providers/dns/factory.go

```diff
@@ -45,10 +45,8 @@ func NewDNSProviderFactory(client client.Client, config depresolver.Config) (f *
 func (f *ProviderFactory) Provider() Provider {
 	a := assistant.NewGslbAssistant(f.client, f.config.K8gbNamespace, f.config.EdgeDNSServers)
 	switch f.config.EdgeDNSType {
-	case depresolver.DNSTypeNS1:
-		return NewExternalDNS(externalDNSTypeNS1, f.config, a)
-	case depresolver.DNSTypeRoute53:
-		return NewExternalDNS(externalDNSTypeRoute53, f.config, a)
+	case depresolver.DNSTypeNS1, depresolver.DNSTypeRoute53:
```

Comment:
Can extend change and we get rid of
`depresolver.DNSTypeNS1, depresolver.DNSTypeRoute53` and have only `depresolver.DNSTypeExternal` ? 
Will require changes in `operator.yaml` as well I guess.

Reply:
Ok, will drop that

Reply:
@kuritka done